### ISSUE TITLE
[bug] importing `@tamagui/next-plugin` prevents node process from exiting

### ISSUE THREAD

**Issue by coffeebeats** [tamagui/tamagui#540]
#### Describe the bug

`@tamagui/next-plugin` is causing my tests to hang when I import `withTamagui` in my `next.config.js` file. The root cause for this is 
a [`setInterval` call in `packages/static/src/extractor/getStaticBindingsForScope.ts`](https://github.com/tamagui/tamagui/blob/6a61d09dee4eebecd27d30c752a12d6674815f02/packages/static/src/extractor/getStaticBindingsForScope.ts#L25-L29).

The `getStaticBindingsForScope.js` file gets imported by `@tamagui/next-plugin` despite not being used (to my knowledge). The following import chain causes this:

1. [Importing multiple properties](https://github.com/tamagui/tamagui/blob/6a61d09dee4eebecd27d30c752a12d6674815f02/packages/next-plugin/src/withTamagui.ts#L9) from `@tamagui/loader`
2. [Importing multiple properties from](https://github.com/tamagui/tamagui/blob/6a61d09dee4eebecd27d30c752a12d6674815f02/packages/loader/src/loader.ts#L3-L7) `@tamagui/static`
3. [Importing `createExtractor`](https://github.com/tamagui/tamagui/blob/6a61d09dee4eebecd27d30c752a12d6674815f02/packages/static/src/index.ts#L4) from `./extractor/createExtractor.js`
4. [Importing two properties](https://github.com/tamagui/tamagui/blob/6a61d09dee4eebecd27d30c752a12d6674815f02/packages/static/src/extractor/createExtractor.ts#L41-L44) from `./getStaticBindingsForScope.js`

#### To Reproduce

The issue can be demonstrated by the following simple example:

_Package.json_
```js

{
  "name": "example",
  "version": "0.0.1",
  "main": "index.js",
  "scripts": {
    "start": "node index.js"
  },
  "license": "MIT",
  "dependencies": {
    "@tamagui/next-plugin": "1.0.11"
  }
}
```

_index.js_

```js
const { withTamagui } = require("@tamagui/next-plugin");
```

With this example installed via `npm install`, the `npm start` process will never exit. If I comment out that `setInterval` call, then it terminates as expected.

#### To Fix

I'm happy to submit a PR to address this, but I was wondering what the preferred solution would be to resolve this?

#### System Info (please complete the following information):

System:
  - OS: Linux 5.15 Ubuntu 20.04.5 LTS (Focal Fossa)
  - CPU: (16) x64 AMD Ryzen 7 3700X 8-Core Processor
  - Memory: 13.06 GB / 15.59 GB
  - Container: Yes
  - Shell: 5.8 - /usr/bin/zsh
 
Binaries:
  - Node: 19.4.0 - ~/.volta/tools/image/node/19.4.0/bin/node
  - Yarn: 3.3.1 - ~/.volta/tools/image/yarn/3.3.1/bin/yarn
  - npm: 9.2.0 - ~/.volta/tools/image/node/19.4.0/bin/npm
 
npmPackages:
  - @tamagui/next-plugin: ^1.0.11 => 1.0.11

**Comment by natew** [comment]
Oh wow, amazing. Yea I’d wondered what’s going on here. We can definitely clear that interval, not as my desk atm but PR happily accepted. Likely process exit listener that clears?

**Comment by natew** [comment]
Fix in next release 42bec0c3db3665f8ca5b3933a8922697bf44ed23
